src/Serialization/Fields.jl

```diff
@@ -5,13 +5,6 @@
 ################################################################################
 # field of rationals (singleton type)
 @register_serialization_type QQField
-# exclude from ring union definition
-type_params(::QQField) = TypeParams(QQField, nothing)
```

Comment:
I am a little confused what's happening with `QQ`  now that this has been removed. For example 

```
julia> Oscar.Serialization.type_params(QQ)
Type parameters for QQField ZZ
```
The printing could be improved... but the point is the return is `ZZ` and not nothing because of 

```
type_params(R::T) where T <: FracField = TypeParams(T, base_ring(R))
```

However, `QQField` still seems to serialize properly including examples when `QQField` is a paramtere.